packages/docs/celo-gold-holder-guide/cloud-hsm.md

```diff
@@ -0,0 +1,101 @@
+# Using a Cloud HSM
+
+A cloud Hardware Security Module (HSM) provides a good balance between security and accessibility. A cloud HSM can manage a Celo private key and can be used seamlessly with `celocli` and `contractkit`. Similar to a ledger device, a key in an HSM is more secure than an in-memory key since the key can never leave the hardware boundary and all signing is performed remotely. To authenticate to the HSM, it's recommended to create a service principal account that has been granted access to sign with the managed keys. A cloud HSM can be a great option for managing voting keys, since you may want these accounts to be portable but still desire to maintain good security practices. This guide will walk you through creating a cloud HSM in Azure and connecting it to `celocli`. 
```

Comment:
This is a good description, the only thing that I have to point here, is that you are saying "Similar to a ledger device, a key in an HSM `is more secure` than an in-memory key since the key can never leave the hardware boundary and all signing is performed remotely."
We both know that that statement is true, but you're applying a value judgment there. And due to we don't really know if HSM would have a mega bug in the future (or it has it now and nobody knows that), this could be thought as the document telling you to use another option (not in memory). Maybe I'm being ultra paranoid, but don't know if this could be a possible legal problem (maybe @pranaymohan could help us with this)

Probably something like:
"Similar to a ledger device, a key in an HSM `avoids using an in-memory key` since the key can never leave the hardware boundary and all signing is performed remotely."
removes that, and everyone that knows what a ledger is, will probably know that is more secure
(I searched all the other texts for the same, and I didn't found another)

Reply:
Totally agree... thanks for the suggestion!